tasks/ceph_manager.py

```diff
@@ -112,6 +113,7 @@ def __init__(self, manager, config, logger=None):
         self.clean_wait = self.config.get('clean_wait', 0)
         self.minin = self.config.get("min_in", 3)
         self.chance_move_pg = self.config.get('chance_move_pg', 1.0)
+        self.hangup_delay = self.config.get('hangup_delay')
```

Comment:
Good call, I won't here but when I use it in time.sleep I will.  I want a return value of None here to be ok.